### PR TITLE
fix(unit test): test_list_all_available_nemesis missed nemesis

### PR DESCRIPTION
**Pull request by juliayakovlev**
One nemesis was missed in data_dir/nemesis.yml.
It caused to failure of `test_list_all_available_nemesis`

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: unit_tests/test_nemesis_sisyphus.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def disrupt(self):
         pass
 
 
-def test_list_all_available_nemesis(generate_file=True):
+def test_list_all_available_nemesis(generate_file=False):
```
Comment (fruch) [inline unit_tests/test_nemesis_sisyphus.py:64]: Why are you changing this ?

**Comment by fruch** [comment]
@juliayakovlev the unittests are still failing

File: unit_tests/test_nemesis_sisyphus.py (hunk 1 above)
Comment (juliayakovlev) [reply unit_tests/test_nemesis_sisyphus.py]: fixed

**Comment by juliayakovlev** [comment]
> @juliayakovlev the unittests are still failing

This is weird as it pass me locally

**Comment by juliayakovlev** [comment]
> @juliayakovlev the unittests are still failing

Now it pass

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**